pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -454,6 +455,13 @@ func setupKernelTunables(option KernelTunableBehavior) error {
 			klog.V(2).InfoS("Updating kernel flag", "flag", flag, "expectedValue", expectedValue, "actualValue", val)
 			err = sysctl.SetSysctl(flag, expectedValue)
 			if err != nil {
+				if libcontainersystem.RunningInUserNS() {
+					if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.KubeletInUserNamespace) {
+						klog.Warningf("Updating kernel flag failed: %v: %v (running in UserNS, ignoring)", flag, err)
```

Comment:
I think that they've already migrated this to the new logging format 
```
				klog.ErrorS(err, "Failed to get container name for process")
```

Reply:
updated, thanks